datatypes/src/operations/image/colorizer.rs

```diff
@@ -550,7 +550,7 @@ impl TryFrom<SerializablePalette> for Palette {
 
 /// `RgbaColor` defines a 32 bit RGB color with alpha value
 #[derive(Copy, Clone, Debug, Deserialize, Serialize, PartialEq, Eq)]
-pub struct RgbaColor([u8; 4]);
+pub struct RgbaColor(pub [u8; 4]);
```

Comment:
Why do we need to make this pub?

Reply:
the RgbaColor is part of our API but defined in the datatypes crate. Thus I copied it to the services crate and implemented a From-Conversion. But that requires access to the map.

Reply:
cf. above

Reply:
4cc60a0b64b1d9fb4b45e664ff15b6cc92a8de3e